kano_profile/tracker/__init__.py

```diff
@@ -1,227 +1,52 @@
-#!/usr/bin/env python
-
-# tracker.py
+#
+# __init__.py
```

Comment:
I would actually move the Tracker to another repo / project as it doesn't have a lot to do with kano-profile itself. More of a general note.

Reply:
I would definitely be for that.